doc/developer.texinfo

```diff
@@ -2446,34 +2436,99 @@ GNU Texinfo Manual (@uref{http://www.gnu.org/software/texinfo/manual/texinfo/tex
-(http://plse.cs.washington.edu/daikon/staging-daikon/).
-Part of this step is to build a set of publication abstracts.  If there are
-any failures, then investigate and correct as necessary.
-The final output of this step
-will be a list of files that were added/removed since the
-last release.
-If any of these differences is unexpected, then investigate and correct the
-difference, and redo the step.
+Once you have successfully reached this point, you are ready to make a release
+candidate and begin testing it. Any source changes required to fix a problem
+encountered above must be committed to the repository now.
+Verify doc/CHANGES is up-to-date and there are no uncommitted files.
+
+@example
+make check-repo
+@end example
+
+@noindent
+The result of the command above should be:
+
+@example
+On branch master
+Your branch is up-to-date with 'origin/master'.
+nothing to commit (use -u to show untracked files)
+@end example
+
+[Time: moments]
+
 
 @need 1000
+@item
+Set the version number in file @file{$DAIKONDIR/doc/VERSION}
+to an even number.
+You may do this manually, or if the value just needs to be incremented
+by 1 (to go from odd to even) you may use:
+
+@c "cd $DAIKONDIR" because the previous commands changed the working directory
 @example
-cd $DAIKONDIR
-\rm -f `find . -type f -name '*~'`
-make staging
+make update-dist-version-file
 @end example
 
-[Time: 4 min]
+Optionally, override the distribution date (default: today) by
+redefining the environment variable @command{TODAY}:
+
+@example
+export TODAY='November 1, 2013'
+@end example
 
+@need 1000
+Now, update the appropriate files with the date and version number
+of the release and commit these changes back to the repository:
 
+@example
+make update-and-commit-version
+@end example
+
+[Time: moments]
+
+
+@need 1000
 @item
-Check the website to see if it has any broken links.
+It is important to build the release candidate from a clean enlistment.
+This is to ensure we are testing only the files intended for release and
+that those files are the same as a user would receive with a clone of
+the repository. This needs to be done in a separate, new Bash command
+shell.
+
+@example
+mkdir -p /tmp/$USER
+rm -rf /tmp/$USER/daikon
+git clone https://github.com/codespecs/daikon /tmp/$USER/daikon
+cd /tmp/$USER/daikon
+export DAIKONDIR=`pwd`
+export JAVA_HOME=/homes/gws/markro/jdk7
+source $DAIKONDIR/scripts/daikon.bashrc
+@end example
 
-@c missing software has been loaded onto buffalo.  (12/2/2014)
-@c Do not run this step on
-@c @samp{buffalo}.  For some reason the system software is not up to date.  Both
-@c @samp{bicycle} and @samp{tricycle} have been verified to work properly.
+@c [Time: 1 minute]
+
+Next, build everything to be distributed and copy it to the temporary staging area
+@c not @uref{...} because we don't want robots to notice and crawl the link
+(http://plse.cs.washington.edu/staging-daikon/).
+
+@example
```

Comment:
How about combining this with the previous `@example` so that users can paste the whole thing at once rather than having to paste, then wait for completion, then paste again?